comp/website/switch-to-normal-mode.yaml

```diff
@@ -1,17 +0,0 @@
-summary: Switch the website back to the normal homepage
```

Comment:
It's in the task with switching the website into live mode. The intention is to collect together the actions related to having the website in comp mode.

Reply:
I think this is a bad idea. The actions are distinct, with separate timelines. We do want to track that each of them has happened. Forgetting to actually put the website into competition mode because the task has to remain open would be bad. Similarly, forgetting to revert the website because the task has been closed would also be bad.

If this were the runbook then I'd agree that having these documented together makes sense. However even in the runbook we have checklists which break tasks out into their components so there's a reference of what needs doing and when.
If runbook tasks were at the level of "Organise the website for the competition" and included everything from getting a venue map on the site to doing the mode swapping then I'd maybe see a case for having these combined, however that's not where the recurring tasks is currently, not the original intent, nor where this PR takes it (and I think we'd agree not very useful either).

A lot of the tasks in this repo (these ones included) are the result of learning from when things went wrong -- usually because we forgot to do things, which were either not documented or insufficiently clearly documented. I am quite concerned that the direction of changes in this PR is ignoring those learnings and risks relying on error-prone human actions rather than allowing the recurring tasks to take the weight.

Reply:
We've updated ([c8d347e](https://github.com/srobo/recurring-tasks/pull/90/commits/c8d347e3e9a8beee1ec83da841cd5cc665bbf86c)) the task to make it explicit that the PRs to switch in and out of competition modes should be added to the project so they can be tracked in the same place as other tasks. Hopefully this should mean that they are not missed as we have just one place to look.

>  I am quite concerned that the direction of changes in this PR is ignoring those learnings and risks relying on error-prone human actions rather than allowing the recurring tasks to take the weight.

This is something I am also concerned about, however I think it's worth acknowledging that a lot of the learnings that were baked into the tasks were from when we ran things in a different way. My hope is that by having area owners and reviewing the tasks for each area as part of the weekly area catchups, we will have visibility over all the tasks and the opportunity to spot something that may need intervention early.